src/corpus/application/TextService.ts

```diff
@@ -1,13 +1,13 @@
-import Bluebird from 'bluebird'
+import Promise from 'bluebird'
```

Comment:
Why rename to `Promise`? Using `Promise` will hide the original JS promise and there will be no error if the import is accidentally removed.

Reply:
i changed it but sometimes we use `Bluebird from "bluebird"` and sometimes `Promise from "bluebird"`